samples/dnn/yolo_object_detection.cpp

```diff
@@ -156,8 +164,9 @@ int main(int argc, char** argv)
                     int baseLine = 0;
                     Size labelSize = getTextSize(label, FONT_HERSHEY_SIMPLEX, 0.5, 1, &baseLine);
                     rectangle(frame, Rect(Point(xLeftBottom, yLeftBottom ),
-                                          Size(labelSize.width, labelSize.height + baseLine)),
-                              Scalar(255, 255, 255), CV_FILLED);
+                              Size(labelSize.width, labelSize.height + baseLine)),
```

Comment:
This indentation change does not correct: `Size()` is argument of `Rect()` ctor, not `rectangle()` call.

Reply:
Okay, but the code was already in the previous version.

I will fix

Reply:
ok, I understand sorry

Reply:
done